exonum-java-binding/core/rust/src/utils/jni_cache.rs

```diff
@@ -90,6 +90,8 @@ unsafe fn cache_methods(env: &JNIEnv) {
         "getErrorCode",
         "()B",
     );
+    RUNTIME_ADAPTER_INITIALIZE =
```

Comment:
Just in case: there is no measurable performance benefit in caching infrequently called methods; so it is not required. But maybe for consistency and clearer code organization it still makes sense.

Reply:
Yep, it's probably an over-engineering by me. Though I'd leave it as it is, just for consistency.